src/main/java/net/rpgtoolkit/common/utilities/TileSetCache.java

```diff
@@ -20,7 +27,7 @@
 public class TileSetCache {
 
   // Singleton.
-  private static final TileSetCache instance = new TileSetCache();
+  private static final TileSetCache INSTANCE = new TileSetCache();

```

Comment:
Since this is already static and already initializing regardless of whether it gets used, can we just make this all static, or is there a benefit to having this be a singleton with an actual instance?